org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -6490,4 +6490,27 @@ public List<PO> get_TableAttributes()
 		return new Query(Env.getCtx(), MTableAttribute.Table_Name, "AD_Table_ID=? AND Record_ID=? ", null).setParameters(get_Table_ID(), get_ID()).list();
 	}
 
+	/** Set of active AD_Workflow_IDs */
+	private final Set<Integer> activeWorkflows = new HashSet<>();
```

Comment:
I've not check in details but in general, please avoid adding instance variable to PO since PO is cacheable.